package.json

```diff
@@ -22,6 +22,7 @@
     "@types/react": "^16.9.0",
     "@types/react-cytoscapejs": "^1.1.0",
     "@types/react-dom": "^16.9.0",
+    "@use-it/event-listener": "^0.1.5",
```

Comment:
The thing is 32 lines, not worth having as a package dependency. Especially if used only in one place.

https://github.com/donavon/use-event-listener/blob/develop/src/index.js

Reply:
Yeah, no problem. It's MIT. I can plop it in manually if you want. I looked at the source and that's why I chose it. Like I mentioned in the original PR comment, I built a much better one from scratch for work that has been battle tested quite a bit and this was the closest I could find. But, I can't use the one I built for work because I don't own it.

Reply:
This actually turned out to be quite a bit of work but I learned a lot about Event types in Typescript. In short, they suck and aren't generic. 

His type definition is actually quite complicated and I honestly couldn't figure out how to incorporate it into the file itself from scratch. 

https://github.com/donavon/use-event-listener/blob/develop/types/index.d.ts

I got pretty close, but still ended up having to cast as EventListener to use the native function.